cypress/integration/branch/experiment.setup.spec.js

```diff
@@ -174,17 +205,6 @@ context("Testing Experiment Selection View", () => {
       cy.wait(5000)
       sensorData.getRecordButton().last().click()
 
-      // TODO: add checks that time series data displays
```

Comment:
Since deleting trial runs is nontrivial, I decided to move this code in its own test in order to do a hard reset.